crates/rpc/src/sequencer.rs

```diff
@@ -0,0 +1,114 @@
+//! Optimism reth RPC Extension used to forward raw transactions to the sequencer.
```

Comment:
just curious, why do we need to forward txs to the sequencer from the rollup node?

Reply:
This is actually for op-reth - starting to move things out of https://github.com/paradigmxyz/reth/blob/main/crates/optimism/rpc/src/eth/rpc.rs. Happy to nix this if you think it's a better idea to wait on moving op-reth stuff over here

Reply:
Actually, think I convinced myself it's worth removing this for now. I'll rework this PR so we can maybe move the RPC across the line.